README.md

```diff
@@ -28,6 +28,14 @@ file. In order to pass the parameters from the example above by means of these c
     echo 1 > ~/AutoRaise.delay
     echo "0.5 0.1 2.5" > ~/AutoRaise.warp
 
+Using of **~/.AutoRaise** or **~/.config/AutoRaise/config** also possible:
```

Comment:
Good that you pay intention to this detail :-)
Might want to rephrase it to something like:

"Update (2021-04-16): In addition to the configuration files mentioned above, AutoRaise now supports hidden configuration files in these locations: **~/.AutoRaise** or **~/.config/AutoRaise/config**. The format of these files is different as well:

Reply:
Ok